### PR TITLE
feat: updated tegel package and changed prefix

### PR DESCRIPTION
**Pull request by timrombergjakobsson**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @import url('@scania/tegel/dist/tegel/tegel.css');
-@import url('https://cdn.digitaldesign.scania.com/icons/webfont/css/sdds-icons.css');
+@import url('https://cdn.digitaldesign.scania.com/icons/webfont/css/tds-icons.css');
```
Comment (timrombergjakobsson) [inline src/index.css:2]: Oops, trigger happy, this is incorrect. Will revert!

File: src/pages/WebComponents.tsx [hunk 2]
```diff
@@ -95,58 +95,58 @@ export default function WebComponents() {
...
+          </tds-block>
+        </tds-block>
       </div>
       <div>
-        <div className="sdds-headline-02 sdds-u-pb1">Breadcrumbs</div>
-        <sdds-breadcrumb>
-          <sdds-breadcrumb-item>
+        <div className="tds-headline-02 tds-u-pb1">Breadcrumbs</div>
+        <tds-breadcrumb>
+          <tds-breadcrumb-item>
```
Comment (mJarsater) [inline src/pages/WebComponents.tsx:105]: All breadcrumb-items needs to be tds-breadcrumb

**Review by mJarsater — APPROVED** [review]
Hope you don't mind me adding a commit. But looks good to me now!

**Comment by timrombergjakobsson** [comment]
> Hope you don't mind me adding a commit. But looks good to me now!

Ofc not! Thanks!

**Pull request merged**